### ISSUE TITLE
Migrating from Gitbook

### ISSUE THREAD

**Issue by cfaddict** [vuejs/vuepress#951]
I am trying to migrate some docs from Gitbook and being brand new to VuePres I am unsure of how to do something. We have these conditionals in our docs that are used for 2 purposes. When we are running the documentation in dev mode (gitbook serve) they only show 1 language. The more important piece is that when we build it we build documentation for a specific language. That means where ever we see these conditionals they need to be evaluated, only show one language and build it into that languages folder (java/.net/etc) 

```
{% if book.language === "Java" %}
``` java
    int numOfBananas = ...;
    String message = 
        "We have " + numOfBananas + " banana" + (numOfBananas != 1 ? "s" : "");
```
{% elif book.language === 'C#' %}
``` csharp
    int numOfBananas = ...;
    string message = 
        "We have " + numOfBananas + " banana" + (numOfBananas != 1 ? "s" : "");
```
{% endif %}
```

I know I can use vue directives and logic in my markdown files but what I am unsure how to do is pass additional options to vuepress dev and vuepress build and where to store those options. Any help here would be greatly appreciated.

**Comment by cfaddict** [comment]
For just the display portion I was able to add a variable in themeConfig for language and this evaluates to true.

**Comment by x8x** [comment]
````
'''java 
code here
'''
````
should be:
````
```java
code here
```
````

**Comment by cfaddict** [comment]
@x8x  Yes sorry for the confusion, I couldn't figure out how to escape them on Github. I can escape them in normal markdown but to post them here on github inside of a code block was an issue.

**Comment by heyheni** [comment]
@x8x have a look at typora i'ts a fantastic markdown editor. https://typora.io
It will create code blocks, tables and [mermaid](https://mermaidjs.github.io) diagramms for you.
[(how to implement mermaid in vuepress)](https://github.com/vuejs/vuepress/issues/111)

**Comment by eunjae-lee** [comment]
@cfaddict If you iterate over all languages and build documents for each language, I think you could pass the language as an environment variable and access it in Vue with `process.env.xxx`.
